cumulusci/tasks/tests/test_pushfails.py

```diff
@@ -67,7 +67,7 @@ def test_run_task(self, *mocks):
                 },
             ],
         }
-        with temporary_dir() as d:
+        with temporary_dir() as temp_dir:
```

Comment:
You don't have to give it a name at all since you're just using it for the side effect and not using it within the with block

Reply:
right, originally i thought I might need it to read the file, but i'm letting the task give me the name anyway.